### DISCUSSION TITLE
Empty transcript when streaming RTP audio

### DISCUSSION THREAD

**Discussion by aashish-joshi** [deepgram/community#756] [General help] [answered]
Hello,

I am streaming RTP audio from Asterisk to Deepgram using the Python SDK. Before sending the audio payload to Deepgram, I remove the header and only send the payload.

I get `SpeechStarted` and `Results` events back, but the transcript is always empty. I am not sure what the issue is. Would it be possible to share insights on what might be wrong?

Here's a sample request ID: `22929702-37d4-473e-88aa-80d5280f41f1`.

The options used to connect are:
model = `nova-2`
encoding = `linear16`
sample_rate = `8000`
vad_events = `True`

Here is a sample response received from Deepgram.

```json
{
    "type": "Results",
    "channel_index": [
        0,
        1
    ],
    "duration": 4.91,
    "start": 43.31,
    "is_final": true,
    "speech_final": false,
    "channel": {
        "alternatives": [
            {
                "transcript": "",
                "confidence": 0.0,
                "words": []
            }
        ]
    },
    "metadata": {
        "request_id": "22929702-37d4-473e-88aa-80d5280f41f1",
        "model_info": {
            "name": "2-general-nova",
            "version": "2024-01-11.36317",
            "arch": "nova-2"
        },
        "model_uuid": "1dbdfb4d-85b2-4659-9831-16b3c76229aa"
    }
}
```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @aashish-joshi, typically this means the sample rate is incorrect. Try using a higher sample rate, such as 16k rather than 8k.

**Reply by aashish-joshi** [reply]
@jjmaldonis Thank you. That seems to have done the trick. :)

**Comment by Thomcle** [comment]
I have an audio which sample rate is 44100 and it doesn't work.
